check_embed_widget.py

```diff
@@ -0,0 +1,17 @@
+import napari
+from skimage.data import astronaut
+from micro_sam.sam_annotator._state import SamState
```

Comment:
Just fyi, I have called this `AnnotatorState` in https://github.com/computational-cell-analytics/micro-sam/pull/240 now.